test/fixtures/sign-output.json

```diff
@@ -1,4 +1,4 @@
 {
-  "signedTransaction": "12000322000000002400000017201B0086955468400000000000000C732102F89EAEC7667B30F33D0687BBA86C3FE2A08CCA40A9186C5BDE2DAA6FA97A37D87446304402207660BDEF67105CE1EBA9AD35DC7156BAB43FF1D47633199EE257D70B6B9AAFBF02207F5517BC8AEF2ADC1325897ECDBA8C673838048BCA62F4E98B252F19BE88796D770A726970706C652E636F6D81144FBFF73DA4ECF9B701940F27341FA8020C313443",
-  "id": "DB44C111583A95AF973A0B0A40348D90512FCBCDDCA3315A286D2BF4FAC100F1"
+  "id": "771B7F94C1E9E0D3B64654799491D6DB06CD02D1C80EE09B2E63B9B3B7AAEF65",
+  "signedTransaction": "12000322000000002400000017201B0086955468400000000000000C732102F89EAEC7667B30F33D0687BBA86C3FE2A08CCA40A9186C5BDE2DAA6FA97A37D874473045022100B3CCDC7DFD929A45B97C55F2E784FFE372A8CE0B2071FC13458F3B2FC0D03DC60220299C1CC39A1A051A9C1C47FAA5210EC6A72D1102CB5F8811CB21DCF60A6319AB770A726970706C652E636F6D81144FBFF73DA4ECF9B701940F27341FA8020C313443"
```

Comment:
Oh, I guess it shouldn't make a difference to rippled because it should work for any arbitrary value of k right?